spec/xclarity_client_update_comp_spec.rb

```diff
@@ -47,19 +47,19 @@
     expect(XClarityClient::VERSION).not_to be nil
   end
 
-  describe 'GET /updatableComponents' do
-    context 'get updateComponents with default action of applyStatus' do
-      it 'should respond with an array' do
-        expect(@client.discover_firmware_update_status).not_to be_empty
-      end
-    end
+  #describe 'GET /updatableComponents' do
```

Comment:
We can not do that! Once those test cases could expose some real failures, impacting directly on this gem clients. We need to investigate more what is causing such failures that are happen in some environments.